dss/config.py

```diff
@@ -92,34 +92,40 @@ def set_config(config: BucketConfig):
         Config._CURRENT_CONFIG = config
 
     @staticmethod
-    def get_cloud_specific_handles(replica: str) -> typing.Tuple[
-            BlobStore, HCABlobStore, str]:
+    def get_cloud_specific_handles(replica: Replica) -> typing.Tuple[BlobStore, HCABlobStore, str]:
```

Comment:
should do the union trick here as well.

Reply:
I'm rolling that back. I think we should force people to use `Replica` or fail fast. This will encourage the usage we want to see.